server/index.js

```diff
@@ -1,18 +1,38 @@
 const express = require('express');
 const cors = require('cors')
+const petModel = require('./db/index');
+const mongoose = require('mongoose');
 
 const app = express();
 const port = 8000;
```

Comment:
in Production the port should come from environment variables. please read about process.env.NODE_ENV

Please see how you pass the NODE_ENV etc in the npm script.

Reply:
Done it with dotenv package